static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.tsx

```diff
@@ -0,0 +1,231 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+import isEqual from 'lodash/isEqual';
+
+import AsyncComponent from 'sentry/components/asyncComponent';
+import BarChart from 'sentry/components/charts/barChart';
+import {DateTimeObject} from 'sentry/components/charts/utils';
+import IdBadge from 'sentry/components/idBadge';
+import {getParams} from 'sentry/components/organizations/globalSelectionHeader/getParams';
+import PanelTable from 'sentry/components/panels/panelTable';
+import Placeholder from 'sentry/components/placeholder';
+import {IconArrow} from 'sentry/icons';
+import {t} from 'sentry/locale';
+import ProjectsStore from 'sentry/stores/projectsStore';
+import space from 'sentry/styles/space';
+import {Organization, Project} from 'sentry/types';
+
+import {
+  barAxisLabel,
+  convertDaySeriesToWeeks,
+  convertDayValueObjectToSeries,
+} from './utils';
+
+type StatusCounts = {
+  resolved?: number;
+  ignored?: number;
+  deleted?: number;
+  new?: number;
+  regressed?: number;
+  unignored?: number;
+  total: number;
+};
+
+type IssuesBreakdown = Record<string, Record<string, StatusCounts>>;
+
+type Statuses = keyof Omit<StatusCounts, 'total'>;
+
+type Props = AsyncComponent['props'] & {
+  organization: Organization;
+  projects: Project[];
+  teamSlug: string;
+  statuses: Statuses[];
+} & DateTimeObject;
+
+type State = AsyncComponent['state'] & {
+  issuesBreakdown: IssuesBreakdown | null;
+};
+
+const keys = ['deleted', 'ignored', 'resolved', 'unignored', 'regressed', 'new', 'total'];
+
+class TeamIssuesBreakdown extends AsyncComponent<Props, State> {
+  shouldRenderBadRequests = true;
+
+  getDefaultState(): State {
+    return {
+      ...super.getDefaultState(),
+      issuesBreakdown: null,
+    };
+  }
+
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    const {organization, start, end, period, utc, teamSlug, statuses} = this.props;
+    const datetime = {start, end, period, utc};
+
+    return [
+      [
+        'issuesBreakdown',
+        `/teams/${organization.slug}/${teamSlug}/issue-breakdown/`,
+        {
+          query: {
+            ...getParams(datetime),
+            statuses,
+          },
+        },
+      ],
+    ];
+  }
+
+  componentDidUpdate(prevProps: Props) {
+    const {start, end, period, utc, teamSlug, projects} = this.props;
+
+    if (
+      prevProps.start !== start ||
+      prevProps.end !== end ||
+      prevProps.period !== period ||
+      prevProps.utc !== utc ||
+      prevProps.teamSlug !== teamSlug ||
+      !isEqual(prevProps.projects, projects)
+    ) {
+      this.remountComponent();
+    }
+  }
+
+  renderLoading() {
+    return this.renderBody();
+  }
```

Comment:
This looks like it's not necessary.

Reply:
there might be an alwaysRenderBody i forgot about